### PR TITLE
v67.2.0

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,83 @@
+# #develop#
+
+Replace this text with a summarized description of this release's contents.
```
Comment (codeman7) [inline CHANGELOG.md]: Should this be updated even if it's a hotfix?

**Review by codeman7 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jverkoey) [reply CHANGELOG.md]: Yep!

File: components/Dialogs/src/MDCAlertController+ButtonForAction.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialDialogs.h"
+
+@interface MDCAlertController (ButtonForAction)
+
+/**
+ Returns an MDCButton associated with the given action. This method might create the button if
+ it no associated button exist for the action yet. Buttons returned by thismethod may not (yet)
```
Comment (galiak11) [inline components/Dialogs/src/MDCAlertController+ButtonForAction.h:21]: tip: typo in comment: remove "it" in sentence.

**Review by galiak11 — APPROVED** [review]

File: components/Dialogs/src/MDCAlertController+ButtonForAction.h (hunk 2 above)
Comment (jverkoey) [reply components/Dialogs/src/MDCAlertController+ButtonForAction.h:21]: Please file a bug to address this in a follow-up. We can't make changes in the release.

Comment (romoore) [reply components/Dialogs/src/MDCAlertController+ButtonForAction.h:21]: also s/thismethod/this method/

**Pull request merged**